backend/wmg/data/transform.py

```diff
@@ -65,18 +65,25 @@ def compute_ordering(cells, root):
 
         ancestor_ids = [a.id for a in ancestors]
 
-        def recurse(node):
+        def recurse(node: Set[str], depth=0):
+
             if node in cells:
-                yield (node)
+
+                cells.remove(node)
+                yield {"id": node, "depth": depth}
+
+                if node != "CL:0000003":
```

Comment:
why are we checking for this particular id?

Reply:
"CL:0000003" is "naive cell", which we use for "unknown" cell types. However "naive cell" is the parent of virtually all other cell types, so we don't want put everything under naive cell.

In other words the CL structure looks like this:

-cell
---naive cell
------all other cells
------ [...]

but we want to make it look this way:
-cell
--naive celll
--all other cells
--[...]